### PR TITLE
Update to latest cardano-ledger-specs

### PR DESCRIPTION
**Pull request by nc6**

### CODE REVIEW

File: ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs [hunk 1]
```diff
@@ -65,6 +65,8 @@ import           Ouroboros.Consensus.Shelley.Protocol
 import           Ouroboros.Consensus.Shelley.Protocol.Crypto (KES)
 import qualified Ouroboros.Consensus.Shelley.Protocol.State as State
 
+import           Debug.Trace
+
```
Comment (mrBliss) [inline ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs:69]: Oops

File: ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs [hunk 2]
```diff
@@ -240,7 +242,7 @@ protocolInfoShelley genesis protVer mbCredentials =
       }
 
     genesisUtxO :: SL.UTxO c
-    genesisUtxO = SL.UTxO $ Map.fromList
+    genesisUtxO = traceShowId $ SL.UTxO $ Map.fromList
```
Comment (mrBliss) [inline ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs:245]: Oops

File: ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs (hunk 1 above)
Comment (nc6) [inline ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs:69]:
```suggestion
```

File: ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs (hunk 2 above)
Comment (nc6) [inline ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs:245]:
```suggestion
    genesisUtxO = SL.UTxO $ Map.fromList
```

**Comment by nc6** [comment]
bors merge

**Comment by nc6** [comment]
bors stop

**Comment by nc6** [comment]
Cancelled because I need to merge the PR in cardano-ledger first

**Comment by mrBliss** [comment]
> Cancelled because I need to merge the PR in cardano-ledger first

Ok, you can squash your commits before merging.

**Comment by nc6** [comment]
bors merge

**Pull request merged**